### ISSUE TITLE
Layer tap key does not repeat

### ISSUE THREAD

**Issue by jameskeim** [makerdiary/python-keyboard#17]
I probably wouldn't have run across this but I use the SPACE key as my "layer 1" Layer Tap Key. In QMK when I use this configuration, I can make the SPACE key auto repeat by tapping it twice and holding it on the second tap. However, the python-keyboard code doesn't recognize this.

I have found a work around that is great for keys like SPACE and ENTER that don't have shifted variants. I put SPACE on "layer 4" and can then SHIFT-SPACE an auto repeat if I need it. I don't have as much need for auto repeating B, D, or ';' but I could see this as being a potential issue in some edge cases.

**Comment by xiongyihui** [comment]
Indeed, the tap-key does not repeat. It's a byproduct of the tap-key feature. We should document it more clearly.
